dev/benchmarks/imitation_game_flutter/lib/main.dart

```diff
@@ -0,0 +1,93 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+void main() {
+  runApp(InfiniteScrollApp());
+}
+
+class InfiniteScrollApp extends StatelessWidget {
+  const InfiniteScrollApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Infinite Scrolling Flutter',
+      home: InfiniteScrollList(),
+    );
+  }
+}
+
+class InfiniteScrollList extends StatefulWidget {
+  const InfiniteScrollList({super.key});
+
+  @override
+  InfiniteScrollListState createState() => InfiniteScrollListState();
+}
+
+class InfiniteScrollListState extends State<InfiniteScrollList> {
+  List<String> items = List.generate(50, (index) => "Hello"); // Initial 50 rows
+  bool isLoadingMore = false;
+  final ScrollController _scrollController = ScrollController();
+  final int _loadMoreThreshold = 5;
+
+  @override
+  void initState() {
+    super.initState();
+    _scrollController.addListener(_scrollListener);
+  }
+
+  @override
+  void dispose() {
+    _scrollController.dispose();
+    super.dispose();
+  }
+
+  void _scrollListener() {
+    if (_scrollController.position.pixels >=
+            _scrollController.position.maxScrollExtent -
+                _loadMoreThreshold *
+                    50 && // 50 is the approximate height of an row
+        !isLoadingMore) {
+      loadMoreItems();
+    }
+  }
```

Comment:
Can you change this code to use [ListView.builder](https://api.flutter.dev/flutter/widgets/ListView/ListView.builder.html)?  This is more complicated and error prone and is supposed to be a point of comparison.  I think that's going to give you better results.